Jenkinsfile_nightly

```diff
@@ -1,7 +1,7 @@
 #!groovy
 
 properties([
-        pipelineTriggers([cron('H 08 * * 1-5')]),
+        pipelineTriggers([cron('00 21 * * *')]),
```

Comment:
You'll need to put this back to 8am, as the perftest env is shutdown each evening, so won't be able to run at 9pm.

Reply:
Changed